### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by yardasol**
## Summary of changes

This PR fixes the docs not deploying. This is due to a mismatch between the dependencies of v0.4.0 and earlier versions, and v0.5.0.

This PR also removes unused sections of the theory manual, and fleshes out a mathematical description of reprocessing in SaltProc

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Required for Merging
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly. 
- [ ] My change is a source code change
  - [ ] I have added/modified tests to cover my changes
  - [ ] I have explained why my change does not require new/modified tests
- [ ] My change is a user-facing change
  - [ ] I have recorded my changes in the changelog for the upcoming release
- [x] My change is exclusively related to the repository (e.g. GitHub actions workflows, PR/issue templates, etc.)
  - x ] I have verified or justified that my change will work as intended.
- [x] All new and existing tests passed.
  - [x] CI tests pass
  - [x] Local tests pass (including Serpent2 integration tests)

## Checklist for Reviewers

Reviewers should use [this link](https://arfc.github.io/manual/guides/pull_requests) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

**Review by ceserz2 — APPROVED** [review]
hey olek, sorry for the late review but everything looks good. Let see if Nathan has anything to add.

File: doc/methods/reprocessing.rst [hunk 1]
```diff
@@ -3,4 +3,162 @@
 Reprocessing Methods
 ====================
 
-Coming soon!
+SaltProc models separations and feeds using three different data structures:
+
+Material flows
+--------------
+A material flow represents a material with a given
+volume, density, temperature, and nuclide composition.
```
Comment (LukeSeifert) [inline doc/methods/reprocessing.rst:11]: Represents the flow of a material?

File: doc/methods/reprocessing.rst [hunk 2]
```diff
@@ -3,4 +3,162 @@
...
+It can also include information such as void fraction. :class:`saltproc.Materialflow`
+objects contain the prior mentioned quantities as attributes, as well as
+methods to perform the following:
+
+1. Add two :class:`saltproc.Materialflow` objects together
+
+   a. The mass of the sum is :math:`m_{1+2} = m_{1} + m_{2}`
+
+   b. The composition of the sum is :math:`N_{1+2} = \frac{N_{1} * m_{1} + N_{2} * m_{2}}{m_{1} + m_{2}}`
```
Comment (LukeSeifert) [inline doc/methods/reprocessing.rst:20]: Consider including in the above paragraph if `N` and `m` are vectors, scalars, etc.

File: doc/methods/reprocessing.rst [hunk 3]
```diff
@@ -3,4 +3,162 @@
...
+A process is an abstraction of chemical separation, extraction, or some
+other removal. Two processes commonly used in \Gls{msr}s include
+extracting metals using gas sparging and filtering. In SaltProc,
+:class:`saltproc.Process` objects contain data and functions to perform their
+associated processing task. At a minimum, a :class:`saltproc.Process` object includes
+the following
+
+1. A mass flow rate, :math:`\dot{m}`, which specifies the mass of fuel salt a process can operate on per unit time
+2. An extraction efficiency, :math:`\epsilon`, for target element(s).
+3. A method to apply the process on a :class:`saltproc.Materialflow` object.
```
Comment (LukeSeifert) [inline doc/methods/reprocessing.rst:40]: Remove periods for consistency with previous lines

File: doc/methods/reprocessing.rst [hunk 4]
```diff
@@ -3,4 +3,162 @@
...
+from time :math:`i` to time :math:`i+1` to get :math:`\mathbf{n}^{j}(i+1)`, where :math:`A` is the
+depletion matrix. This sequence is called a **depletion step**.
+
+Now, let the mass and volume of depletable material :math:`j` be
+:math:`m^{j}` and :math:`V^{j}` respectively. Let the mass flow rate of process :math:`p` be
+:math:`\dot{m}_{p}`. At the end of each depletion step, SaltProc constructs process
+paths from the process graph defined in the DOT file, and sequentially applies
+each process :math:`p` in each path :math:`r` to the relevant materials to obtain through
+and waste streams for each material. SaltProc tracks the mass and nuclide vector for both through and waste streams. For through
+streams, SaltProc also tracks the volume and mass flow rate. For every node
```
Comment (LukeSeifert) [inline doc/methods/reprocessing.rst:83]: "For" can be removed here I think, unless you mean `through streams`. Consider changing to `through-streams` for readability

File: doc/methods/reprocessing.rst [hunk 5]
```diff
@@ -3,4 +3,162 @@
...
+
+.. math::
+
+    m^{j}_{\text{waste, }p,r} = \alpha_{p,r} m^{j}_{\text{through, }p-1,r} \langle\mathbf{1},\mathbf{n}^{j}_{\text{waste, }p,r}\rangle
+
+
+SaltProc does not currently track the volume and mass flow rate of waste streams.
+In practice, since it is extremely resource intensive to separate individual
+isotopes, SaltProc only allows extraction efficiencies to be defined for
+elements. So, for any isotope :math:`a` of xenon,
```
Comment (LukeSeifert) [inline doc/methods/reprocessing.rst:133]: You could also mention here that for MSRs, chemical processes operate on elements, so having isotope specific extraction efficiencies is not required

**Review by LukeSeifert — APPROVED** [review]
Looks good! I made a few suggestions, but only minor things. Let me know if you'd like me to merge.

**Comment by yardasol** [comment]
@LukeSeifert it's been a hot minute but I just addressed your comments. If this is good to go lmk.

**Review by LukeSeifert — APPROVED** [review]
Looks good!

**Pull request merged**